src/test/java/com/productdock/library/search/data/provider/KafkaTestProducer.java

```diff
@@ -0,0 +1,27 @@
+package com.productdock.library.search.data.provider;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.productdock.library.search.cosumer.messages.InsertBook;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.kafka.core.KafkaTemplate;
+import org.springframework.stereotype.Component;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+@Component
+public class KafkaTestProducer {
+
+    private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+
+    @Autowired
+    private KafkaTemplate<String, String> kafkaTemplate;
+
+    public void send(String topic, InsertBook payload) {
+        String message = "";
+        try {
+            message = OBJECT_MAPPER.writeValueAsString(payload);
+        } catch (JsonProcessingException e) {
+            e.printStackTrace();
```

Comment:
Don't hide this exception here, it is better to let test to fail if  message couldn't be parsed. Otherwise producer will send empty message which is not what test would be expecting.
Just add Exception to method signiture